src/function/open-referral-function/src/FamilyHubs.OpenReferral.Function/ClientServices/HsdaApiService.cs

```diff
@@ -0,0 +1,66 @@
+using System.Net;
+using FamilyHubs.OpenReferral.Function.Entities;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
```

Comment:
Noticed we're using the `Newtonsoft.Json` package, but it's preferable these days to use Microsoft's own `System.Text.Json` library where possible :) Could we switch to that?

https://learn.microsoft.com/en-us/dotnet/api/system.text.json?view=net-8.0

Reply:
Didn't think there was an equivalent mapping for `JArray` and `JObject` but it looks like there might be on closer look, I'll have a look at converting!

Reply:
Switched : https://github.com/DFE-Digital/fh-services/pull/63/commits/10e179ef3507f939962382bcb165d6f5eaeedd4b